trunk/SUAVE/Attributes/Solids/BiCF.py

```diff
@@ -0,0 +1,31 @@
+## @ingroup Attributes-Solids
+
+# B[a-z]_[a_z]F.py
```

Comment:
interesting name

Reply:
Changed to correct title. Error was caused by mistaken application of Wing IDE Regex Search and Replace.